ddl/owner_manager.go

```diff
@@ -174,29 +178,38 @@ func (m *ownerManager) campaignLoop(ctx goctx.Context, key string, wg *sync.Wait
 			continue
 		}
 
-		// Get owner information.
-		resp, err := elec.Leader(ctx)
+		ownerKey, err := GetOwnerInfo(ctx, elec, key, m.ddlID)
 		if err != nil {
-			// If no leader elected currently, it returns ErrElectionNoLeader.
-			log.Infof("[ddl] failed to get leader, err %v", err)
-			continue
-		}
-		leader := string(resp.Kvs[0].Value)
-		log.Infof("[ddl] %s ownerManager is %s, owner is %v", key, m.ddlID, leader)
-		if leader == m.ddlID {
-			m.setOwnerVal(key, true)
-		} else {
-			log.Warnf("[ddl] ownerManager %s isn't the owner", m.ddlID)
 			continue
 		}
+		m.setOwnerVal(key, true)
 
-		m.watchOwner(ctx, string(resp.Kvs[0].Key))
+		m.watchOwner(ctx, ownerKey)
 		m.setOwnerVal(key, false)
 	}
 }
 
+// GetOwnerInfo gets the owner information.
+// And extracting it into a function can improve the test coverage.
```

Comment:
"And extracting it into a function can improve the test coverage."  This is not a proper comment.